drivers/input/Kconfig.tsc_keys

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2024 Arif Balik <arifbalik@outlook.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config INPUT_TSC_KEYS
```

Comment:
Question to @fabiobaltieri: Shouldn't symbol contain "STM32" mention (`INPUT_STM32_TSC_KEYS`) ?
It seem we already have `INPUT_ESP32_TOUCH_SENSOR`.
IMO, it is important to differentiate generic drivers from vendor specific ones.
I don't request the whole `drivers/input/input_tsc_keys.c` implementation should be reviewed to add stm32 in every structure, but maybe the driver naming would deserved to be more specific.